tests/storage/restricted_namespace_cloning/test_restricted_namespace_cloning.py

```diff
@@ -104,140 +47,122 @@ def test_unprivileged_user_clone_same_namespace_negative(
-            {
-                PERMISSIONS_DST: DATAVOLUMES_AND_DVS_SRC,
-                VERBS_DST: CREATE_DELETE_LIST_GET,
-            },
-            {"dv_name": "cnv-2689"},
-            marks=pytest.mark.polarion("CNV-2689"),
-            id="src_ns: dv and dv/src, verbs: create, delete. dst: dv and dv/src, verbs: create, delete, list, get.",
-        ),
-        pytest.param(
+            ADMIN_NAMESPACE_PARAM,
             DV_PARAMS,
             {PERMISSIONS_SRC: DATAVOLUMES_AND_DVS_SRC, VERBS_SRC: ALL},
             {PERMISSIONS_DST: DATAVOLUMES_AND_DVS_SRC, VERBS_DST: ALL},
             {"dv_name": "cnv-2692"},
+            {"verify_image_permissions": True},
             marks=pytest.mark.polarion("CNV-2692"),
-            id="src_ns: dv and dv/src, verbs: *. dst: dv and dv/src, verbs: *.",
-        ),
-        pytest.param(
-            DV_PARAMS,
-            {PERMISSIONS_SRC: DATAVOLUMES_AND_DVS_SRC, VERBS_SRC: ALL},
-            {PERMISSIONS_DST: DATAVOLUMES, VERBS_DST: ALL},
-            {"dv_name": "cnv-2805"},
-            marks=pytest.mark.polarion("CNV-2805"),
-            id="src_ns: dv and dv/src, verbs: *. dst: dv, verbs: *.",
-        ),
-        pytest.param(
-            DV_PARAMS,
-            {PERMISSIONS_SRC: DATAVOLUMES_AND_DVS_SRC, VERBS_SRC: CREATE_DELETE},
-            {PERMISSIONS_DST: DATAVOLUMES, VERBS_DST: CREATE_DELETE_LIST_GET},
-            {"dv_name": "cnv-2808"},
-            marks=pytest.mark.polarion("CNV-2808"),
-            id="src_ns: dv and dv/src, verbs: create, delete. dst: dv, verbs: create, delete, list, get.",
+            id="src_dv_and_dv_source_all_dest_dv_and_dv_source_all",
         ),
         pytest.param(
+            ADMIN_NAMESPACE_PARAM,
             DV_PARAMS,
             {PERMISSIONS_SRC: DATAVOLUMES_SRC, VERBS_SRC: CREATE},
             {PERMISSIONS_DST: DATAVOLUMES, VERBS_DST: CREATE_DELETE_LIST_GET},
             {"dv_name": "cnv-2971"},
+            {"verify_image_permissions": False},
             marks=pytest.mark.polarion("CNV-2971"),
-            id="src_ns: dv/src, verbs: create. dst: dv, verbs: create, delete, list, get.",
+            id="src_dv_source_create_dest_dv_create_delete_list_get",
         ),
     ],
     indirect=True,
 )
-def test_user_permissions_positive(dv_destination_cloned_from_pvc, unprivileged_client):
+def test_user_permissions_positive(
+    unprivileged_client,
+    storage_class_matrix__module__,
+    storage_class_name_scope_module,
+    permissions_pvc_destination,
+    dv_destination_cloned_from_pvc,
+    requested_verify_image_permissions,
+):
     verify_snapshot_used_namespace_transfer(cdv=dv_destination_cloned_from_pvc, unprivileged_client=unprivileged_client)
-    with create_vm_from_dv(dv=dv_destination_cloned_from_pvc):
-        return
+    if (
+        requested_verify_image_permissions
+        and storage_class_matrix__module__[storage_class_name_scope_module]["volume_mode"] == DataVolume.VolumeMode.FILE
+    ):
```

Comment:
If this condition is not met, the test would simply pass without any actual call to create_vm_and_verify_image_permission(). Is that what you want?